app/bundles/ApiBundle/Controller/ClientController.php

```diff
@@ -463,4 +452,9 @@ public function deleteAction(Request $request, $objectId)
             )
         );
     }
+
+    public function getModelName(): string
+    {
+        return 'api.client';
+    }
```

Comment:
```suggestion
```
One more cleanup.

Reply:
We can't do that, the method is abstract on the base class.